_episodes/01-intro.md

```diff
@@ -92,6 +92,15 @@ typing will appear.
 The cursor is usually a flashing or solid block, but it can also be an underscore or a pipe.
 You may have seen it in a text editor program, for example.
 
+Note that your prompt might look a little different. In particular, most popular shell environments by default put your user name and the host name before the `$`. Such a prompt might look like, e.g.:
+
+~~~
+nelle@localhost $
+~~~
+{: .language-bash}
+
+The prompt might even include more than this. Do not worry if your prompt is not just a short `$ `. This lesson does not depend on this additional information and it should also not get in your way. The only important item to focus on is the `$ ` character itself and we will see later why.
+
```

Comment:
Ensure lines are not too long.
```suggestion
Note that your prompt might look a little different. In particular, most popular shell
environments by default put your user name and the host name before the `$`. Such
a prompt might look like, e.g.:

~~~
nelle@localhost $
~~~
{: .language-bash}

The prompt might even include more than this. Do not worry if your prompt is not
just a short `$ `. This lesson does not depend on this additional information and it
should also not get in your way. The only important item to focus on is the `$ `
character itself and we will see later why.

```

Reply:
In an online environment, participants can share screens to see a variety of prompts.  It may be more helpful
to ask about different types of shell environments, and explain that they can be configured to the users
preferences.